curriculum/challenges/english/25-front-end-development/quiz-web-standards/66f1a0fcab4dfe9fcdcb269d.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+Which protocol is responsible for domain name resolution?
 
 #### --distractors--
 
-Placeholder distractor 1
+HTTP
 
 ---
 
-Placeholder distractor 2
+FTP
 
 ---
 
-Placeholder distractor 3
+SMTP
 
 #### --answer--
 
-Placeholder answer
+DNS
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the 404 HTTP response code indicate?
 
 #### --distractors--
 
-Placeholder distractor 1
+Server error
 
 ---
 
-Placeholder distractor 2
+Unauthorized access
 
 ---
 
-Placeholder distractor 3
+Redirect
 
 #### --answer--
 
-Placeholder answer
+Resource not found
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the full form of TCP in TCP/IP?
 
 #### --distractors--
 
-Placeholder distractor 1
+Transfer Control Protocol
 
 ---
 
-Placeholder distractor 2
+Transmission Connection Protocol
 
 ---
 
-Placeholder distractor 3
+Transfer Connection Protocol
 
 #### --answer--
 
-Placeholder answer
+Transmission Control Protocol
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is a static file type?
```

Comment:
Hi Anna, 
The criteria to judge whether or not something is a static file type in that question is static files are delivered to the client exactly as they are stored on the server, without undergoing any server-side processing or modification.
That's why, HTML is chosen as a static file over PHP because HTML is served as it is, while PHP requires server-side execution to generate content.
hope this clarifies your query!